### PR TITLE
Add `wp vip migration dbdelta` command.

### PR DESCRIPTION
**Pull request by joshbetz**

### CODE REVIEW

**Comment by joshbetz** [comment]
Output looks like:

```
$ wp vip migration dbdelta
Added index wp_postmeta KEY `vip_meta_key_value` (`meta_key`(191),`meta_value`(100))
Success: 1 changes.

$ wp vip migration dbdelta
Success: No changes.
```

File: wp-cli/vip-migrations.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+				$site_ids[] = $blog->site_id;
+				$url = $blog->domain . $blog->path;
+				$cmd = "--url={$url} vip migration dbdelta";
+				if ( $dry_run ) {
+					$cmd .= ' --dry-run';
+				}
+
+				WP_CLI::line();
+				WP_CLI::line( WP_CLI::colorize( '%mUpdating:%n ' ) . $blog->domain . $blog->path );
+				WP_CLI::runcommand( $cmd );
```
Comment (mjangda) [inline wp-cli/vip-migrations.php]: Do we need to pass `$args` and `$assoc_args` or are those sent through automagically by `runcommand`?

File: wp-cli/vip-migrations.php [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+		if ( empty( $changes ) ) {
+			WP_CLI::success( 'No changes.' );
+			return;
+		}
+
+		foreach( $changes as $change ) {
+			WP_CLI::line( $change );
+		}
+
+		WP_CLI::success( count( $changes ) . ' change(s).' );
```
Comment (mjangda) [inline wp-cli/vip-migrations.php]: Minor:

```
$count = count( $changes );
WP_CLI::success( _n( '%s change', '%s changes', $count ), number_format_i18n( $count ) ) );
```

File: wp-cli/vip-migrations.php (hunk 1 above)
Comment (joshbetz) [inline wp-cli/vip-migrations.php]: The command includes them above: https://github.com/Automattic/vip-go-mu-plugins/pull/494/files#diff-2721972cae24121aeef8bf02ef7a0e2bR46

Comment (mjangda) [inline wp-cli/vip-migrations.php]: Sorry, I meant the `[<tables>]` arg. I guess only `blog` would be applicable here (https://developer.wordpress.org/reference/functions/wp_get_db_schema/) so we should explicitly set that.

File: wp-cli/vip-migrations.php [hunk 3]
```diff
@@ -37,13 +37,18 @@ function dbdelta( $args, $assoc_args ) {
...
 			foreach( $it as $blog ) {
-				$total++;
-				$site_ids[] = $blog->site_id;
 				$url = $blog->domain . $blog->path;
-				$cmd = "--url={$url} vip migration dbdelta blog";
+				$cmd = "--url={$url} vip migration dbdelta";
+
+				// Update global tables if this is the main site
+				// otherwise only update the given blog's tables
+				if ( is_main_site( $blog->site_id ) ) {
```
Comment (mjangda) [inline wp-cli/vip-migrations.php]: Should `site_id` be `blog_id` here?

Comment (joshbetz) [inline wp-cli/vip-migrations.php]: Yes, good catch.

**Comment by joshbetz** [comment]
r89570-deploy

**Pull request merged**